src/ssh-client-v2.html

```diff
@@ -21,22 +22,27 @@
 </script>
 
 <script type="text/x-red" data-template-name="ssh-client-v2">
+    <div class="form-row">
+        <label for="node-input-name"><i class="icon-tag"></i> Name</label>
+        <input type="text" id="node-input-name" placeholder="Name">
+    </div>
     <div class="form-row">
         <label for="node-input-debug"><i class="fa fa-server"></i> Debug</label>
-        <input type="checkbox" id="node-input-debug" placeholder="">
+        <input type="checkbox" id="node-input-debug" placeholder="" style="display:inline-block; width:auto; vertical-align:top;">
```

Comment:
This change makes the checkbox left-aligned, instead of centered. This is how the checkboxes on the default nodes look.